templates/cluster-template-clusterclass.yaml

```
@@ -498,6 +503,13 @@ spec:
           port:
             type: integer
         type: object
+  - name: apiServerSigningCertExtraCertSANs
+    required: true
+    schema:
+      openAPIV3Schema:
+        description: Set extra Subject Alternative Names (SANs) for the API Server
+          signing certificate.
+        type: string
```

And then you can make this an optional list.
```suggestion
  - name: apiServerSigningCertExtraCertSANs
    schema:
      openAPIV3Schema:
        description: Set extra Subject Alternative Names (SANs) for the API Server
          signing certificate.
        type: array
        items:
          type: string
```